bikespace_api/bikespace_api/api/submissions.py

```diff
@@ -10,6 +10,7 @@
 
 submissions_blueprint = Blueprint("submissions", __name__)
 
+DEFAULT_PAGE_LIMIT = 10
```

Comment:
I think we can go default as 100 for now, since we're not experiencing much traffic

Reply:
Changed to 100

Reply:
I'm not seeing the update, it still shows 10 for me